Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -741,7 +741,7 @@ UpdateFspConfig (
 
     // TSN feature support
     if (IsPchS () || IsPchN () ) {
```

Comment:
This is a little different, Kalp. At some point RPL-S FSP changed the TSN enable to per-port instead of global, so it was different from all the rest of the platforms. This is a difference in the FSP header files, so it needs a build time check. Now RPL-P FSP has updated to include this too.